deployer.sh

```
@@ -27,6 +27,7 @@ ln -snfv $PWD/claude/statusline.sh $HOME/.claude/statusline.sh
 ln -snfv $PWD/claude/commands $HOME/.claude/commands
 ln -snfv $PWD/claude/agents $HOME/.claude/agents
 ln -snfv $PWD/claude/hooks $HOME/.claude/hooks
+ln -snfv $PWD/claude/rules $HOME/.claude/rules
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)